docs/faq.rst

```diff
@@ -104,14 +104,35 @@ You can clean snapshots by using :doc:`nodetool clearsnapshot </operating-scylla
   
 Features
 --------
-I want to try out new features.  How do I enable experimental mode?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You need to add the line :code:`experimental: true`  to your :code:`scylla.yaml` file.
 
-1. Launch the file in a text editor: :code:`$ vi /etc/scylla/scylla.yaml`. (Alternately, on docker, it's :code:`$ docker exec -it your_node vi /etc/scylla/scylla.yaml`);
-2. Add the line :code:`experimental: true`;
+I want to try out new features.  How do I enable them?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ScyllaDB features can be enabled individually either through the configuration file or via command-line flags.
+
+To configure using :code:`scylla.yaml` file:
+
+1. Open the file in a text editor:
+
+   On Linux systems
+
+   :code:`$ vi /etc/scylla/scylla.yaml`
+
+   On Docker
+
+   :code:`$ docker exec -it your_node vi /etc/scylla/scylla.yaml`
```

Comment:
Do we have "vi" installed on our docker? :-)

Reply:
good question, i am not sure. this command line was inherited from 6e97b83b6028326f00b823c4f8f2da4fa5c50d3c . probably we should revisit it later on. @dkropachev do you happen to known if our docker image has `vi` installed?